packages/plugin-connector/src/data-sources/get-datasource-plugins.js

```diff
@@ -0,0 +1,51 @@
+import { discoverEvidencePlugins } from '../plugin-discovery';
+import { buildConnector } from './build-connector';
+import chalk from 'chalk';
+import path from 'path';
+/**
+ * @param {EvidenceConfig} [cfg]
+ * @param {PackageDiscoveryResult} [discoveries] Optional: Pass in already discovered plugins
+ * @returns {Promise<PluginDatabases>}
+ */
+export async function getDatasourcePlugins(cfg, discoveries) {
+	const pluginDiscoveries = discoveries ?? (await discoverEvidencePlugins());
+
+	return await pluginDiscoveries.databases.reduce(
+		/**
+		 * Adds a plugin to a map of EvidencePluginPackages with a corresponding DatabaseConnectorFactory,
+		 * ensuring that no duplicate databases are added.
+		 * @param {Promise<Record<string, { package: EvidencePluginPackage<EvidenceDatabasePackage>, factory: DatabaseConnectorFactory }>>} _acc - A promise representing the current state of the package map
+		 * @param {EvidencePluginPackage<EvidenceDatabasePackage>} v - The plugin package to be added to the map
+		 * @returns {Promise<Record<string, { package: EvidencePluginPackage<EvidenceDatabasePackage>, factory: DatabaseConnectorFactory }>>} - A promise representing the updated package map
+		 */
+		async (_acc, v) => {
+			// TODO: Handle Overrides
+
+			// Wait for the current state of the package map to resolve
+			const acc = await _acc;
```

Comment:
What's an `acc`?

Reply:
accumulator

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#parameters

Reply:
Pretty common in identifier in reduce / fold type functions.